src/features/user/TreeMapper/Analytics/components/Container/index.module.scss

```diff
@@ -1,27 +1,27 @@
 @import '../../../../../../theme/theme';
 
 .container {
-    box-shadow: 0px 0px 20px -10px rgba(0,0,0,0.35);
+  box-shadow: 0px 0px 20px -10px rgba(0, 0, 0, 0.35);
 }
 
 .headerFlexRow {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 30px;
-    border-radius: 10px 10px 0 0;
-    background-color: $primaryLightColor;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 30px;
+  border-radius: 10px 10px 0 0;
+  background-color: $dsWhiteTransparent10;
```

Comment:
As a result of this change, the rounded borders are no longer visible. If we want them to visible, you need an opaque color.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2592/commits/a5587bc1fc19939d23e352fcd66e010cbd62426c